core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/add/CsvToElements.java

```diff
@@ -38,18 +45,21 @@
 @JsonPropertyOrder(value = {"class", "filename"}, alphabetic = true)
 @Since("2.0.0")
 @Summary("Adds elements from a openCypher CSV file")
```

Comment:
Is this right? The class is called `CsvToElements` and not `openCypherCsvToElements`.